gitlab/resource_gitlab_project_freeze_period_test.go

```diff
@@ -0,0 +1,167 @@
+}
+
+func TestAccGitlabProjectFreezePeriod_import(t *testing.T) {
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGitlabProjectFreezePeriodConfig(rInt),
+			},
+			{
+				ResourceName:      "gitlab_project_freeze_period.schedule",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckGitlabProjectFreezePeriodExists(n string, freezePeriod *gitlab.FreezePeriod) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not Found: %s", n)
+		}
+
+		projectID, freezePeriodID, err := projectIDAndFreezePeriodIDFromID(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		conn := testAccProvider.Meta().(*gitlab.Client)
+		gotFreezePeriod, _, err := conn.FreezePeriods.GetFreezePeriod(projectID, freezePeriodID)
+		if err != nil {
+			return err
+		}
+
+		*freezePeriod = *gotFreezePeriod
+
+		return nil
+	}
+}
+
+type testAccGitlabProjectFreezePeriodExpectedAttributes struct {
+	FreezeStart  string
+	FreezeEnd    string
+	CronTimezone string
+}
+
+func testAccCheckGitlabProjectFreezePeriodAttributes(freezePeriod *gitlab.FreezePeriod, want *testAccGitlabProjectFreezePeriodExpectedAttributes) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if freezePeriod.FreezeStart != want.FreezeStart {
+			return fmt.Errorf("got freeze_start %q; want %q", freezePeriod.FreezeStart, want.FreezeStart)
+		}
+		if freezePeriod.FreezeEnd != want.FreezeEnd {
+			return fmt.Errorf("got freeze_end %q; want %q", freezePeriod.FreezeEnd, want.FreezeEnd)
+		}
+
+		if freezePeriod.CronTimezone != want.CronTimezone {
+			return fmt.Errorf("got cron_timezone %q; want %q", freezePeriod.CronTimezone, want.CronTimezone)
+		}
+
+		return nil
+	}
+}
+
+func testAccGitlabProjectFreezePeriodConfig(rInt int) string {
+	return fmt.Sprintf(`
+resource "gitlab_project" "foo" {
+  name = "foo-%d"
+  description = "Terraform acceptance tests"
+
+  # So that acceptance tests can be run in a gitlab organization
+  # with no billing
+  visibility_level = "public"
+}
+
+resource "gitlab_project_freeze_period" "schedule" {
+	project_id = gitlab_project.foo.id
+	freeze_start = "0 23 * * 5"
+	freeze_end =  "0 7 * * 1"
+	cron_timezone = "UTC"
+}
+	`, rInt)
+}
+
+func testAccGitlabProjectFreezePeriodUpdateConfig(rInt int) string {
+	return fmt.Sprintf(`
+resource "gitlab_project" "foo" {
+  name = "foo-%d"
+  description = "Terraform acceptance tests"
+
+  # So that acceptance tests can be run in a gitlab organization
+  # with no billing
+  visibility_level = "public"
+}
+
+resource "gitlab_project_freeze_period" "schedule" {
+  project_id = gitlab_project.foo.id
+  freeze_start = "0 23 * * 5"
+  freeze_end =  "0 7 * * 3"
+  cron_timezone = "UTC"
```

Comment:
Can you modify `freeze_start` and `cron_timezone` from their original values as well?